src/test/integration/fileTest/compare/data/soot-results.csv

```diff
@@ -1 +1 @@
-project;class;method;merge commit;left right dataflow;right left dataflow;left right tainted;right left tainted;left right svfa;right left svfa;confluence
+project;class;method;merge commit;left right tainted;right left tainted;left right svfa;right left svfa;confluence-tainted
```

Comment:
It's ok. When the Overriding analysis has been integrated, it can now run: D

Reply:
Wouldn't it be nice to add that soon? merges are about to be accepted. It should take the same time as this merge here.

Reply:
I think It would be better to integrate this PR and when a version os generated with those algorithms que can create another.